src/test/java/de/tum/in/www1/artemis/util/LocalRepository.java

```diff
@@ -85,7 +85,7 @@ public void configureRepos(String localRepoFileName, Path originRepositoryFolder
         Path filePath = localRepoPath.resolve("test.txt");
         Files.createFile(filePath);
         localGit.add().addFilepattern("test.txt").call();
-        localGit.commit().setMessage("Initial commit").call();
+        localGit.commit().setSign(false).setMessage("Initial commit").call();
```

Comment:
Let's use the extracted method here as well

Reply:
Done